Formula/h/httpie.rb

```diff
@@ -19,10 +19,13 @@ class Httpie < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "21b503294e029136dd6aec1d942e7dc3e7481d969630ae093e4df3be8f812d82"
   end
 
-  depends_on "pygments"
-  depends_on "python-certifi"
```

Comment:
I think there may have been discussions around keeping `certifi` unvendored?

Reply:
oops, thanks for pointing that out